src/styles/menu.scss

```diff
@@ -22,6 +22,10 @@
       }
     }
 
+    &--subpage {
+      margin-left: var(--amplify-space-small);
```

Comment:
This is really just a nit: What do you think about putting this indent on the padding on the link? That way they can still have the full width highlight effect?

Reply:
Thanks Heather, I think with padding-left we don't get the full width highlight effect either unfortunately. I think it's because of how I laid out the html. I can look into this though to see if I can get the highlight effect working

Reply:
Just pushed up the change for this